src/plots/Volume/VolumeAttributes.code

```diff
@@ -654,5 +652,3 @@ VolumeAttributes_SetMaterialProperties(PyObject *self, PyObject *args)
 }
 
 Target: xml2cmake
-Condition: VISIT_SLIVR
```

Comment:
These two lines should have been added back in with the `Target: xml2cmake` line above. I apologize if I wasn't clear in my previous review and subseqent comments.

Reply:
Ok...should've clarified before updating.